packages/ui/src/components/PostScreenView.tsx

```diff
@@ -96,7 +96,9 @@ export function PostScreenView({
               <KeyboardAvoidingView
                 //TODO: Standardize this component, account for tab bar in a better way
                 behavior={Platform.OS === 'ios' ? 'padding' : 'height'}
-                // keyboardVerticalOffset={70}
+                keyboardVerticalOffset={
+                  Platform.OS === 'ios' ? -bottom : DEFAULT_MESSAGE_INPUT_HEIGHT
```

Comment:
Does it make sense to extract this wrapper component and use across multiple screens? I'm guessing we have similar in other places?

Reply:
Looked for that, afaict this is the only place with this exact offset.